coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/log/BaseRestLogger.java

```diff
@@ -57,8 +57,9 @@
     @ThisLogger
     private AppLogger log;
 
-    @Resource(lookup = "java:app/AppName")
-    private String applicationName;
+    @Inject
+    @ConfigProperty(name = "service.name")
```

Comment:
Ennek a konfig kulcsnak a bevezeteset nem lehet elvarni hogy mindenki megtegye. Kell szamolni valami alapertelmezett ertekkel is, majd a coffee konfig parametereibe is beilleszteni, jelen esetben "coffee." prefixet hasznalnak, beszeljunk a megoldasokrol...